IPython/html/base/handlers.py

```diff
@@ -138,6 +138,10 @@ def base_url(self):
         return self.settings.get('base_url', '/')
 
     @property
+    def default_url(self):
+        return self.settings.get('default_url', '/')
```

Comment:
I guess it should be `/tree`. The fallbacks here can never be encountered in IPython, only things like the the single-cell server which reuse a subset of the handlers, and different initialization.